modules/custom/alternative_frontpage/src/EventSubscriber/RedirectHomepageSubscriber.php

```diff
@@ -25,18 +27,11 @@ class RedirectHomepageSubscriber implements EventSubscriberInterface {
   protected $userData;
 
   /**
-   * Protected var alternativeFrontpageSettings.
+   * Protected var configFactory.
```

Comment:
maybe better description like `Configuration Factory` to perform operations on the configs.

Reply:
Will fix it.